hathor/p2p/protocol.py

```diff
@@ -410,6 +411,27 @@ def disable_sync(self) -> None:
         self.log.info('disable sync')
         self.state.sync_agent.disable_sync()
 
+    def is_synced(self) -> bool:
```

Comment:
Missing docstring in all new methods in this file.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1165/commits/b9b84ea0802ea3ae98dbc324d8ab07434a2f3575

Reply:
Thanks!